web/src/index.js

```diff
@@ -385,6 +386,22 @@ class App extends Component {
     }
 
     return html`<main class="has-content ${theme}">
+        <style>
+            a.tab {
+                padding: 5% 5%;
+                width: 40%;
+                text-align: center;
+                border: none;
+                background-color: #f0f0f0;
+                cursor: pointer;
+                -webkit-appearance: button;
+                -moz-appearance: button;
+                appearance: button;
+
+                text-decoration: none;
+                color: initial;
+        }
+        </style>
```

Comment:
I added the CSS code inline here, because I couldn't figure out where else to put it. Probably a code smell, need to fix this.

Reply:
Styles are put in `index.sass` (which is then compiled to update `index.css`)

Reply:
Added the same things to index.sass